apis/swagger.yml

```diff
@@ -1193,24 +1193,34 @@ definitions:
     type: "object"
     properties:
       Name:
-        description: "The new volume's name. If not specified, Docker generates a name."
+        description: "The new volume's name. If not specified, Pouch generates a name."
         type: "string"
-        x-nullable: false
+        minLength: 1
+        maxLength: 64
+        pattern: "[a-zA-Z0-9][a-zA-Z0-9-_.]{0,63}"
       Driver:
         description: "Name of the volume driver to use."
         type: "string"
         default: "local"
-        x-nullable: false
+        minLength: 1
+        maxLength: 64
+        pattern: "[a-zA-Z0-9][a-zA-Z0-9-_.]{0,63}"
       DriverOpts:
         description: "A mapping of driver options and values. These options are passed directly to the driver and are driver specific."
         type: "object"
         additionalProperties:
           type: "string"
+          minLength: 1
+          maxLength: 128
+          pattern: "[a-zA-Z0-9][a-zA-Z0-9-_.]{0,127}"
       Labels:
-        description: "User-defined key/value metadata."
+        description: "User-defined key/value metadata. White space could exist in value."
         type: "object"
         additionalProperties:
           type: "string"
+          minLength: 1
+          maxLength: 128
+          pattern: "[a-zA-Z0-9][a-zA-Z0-9-_. ]{0,127}"
```

Comment:
I do not think this is correct. Since in some environment, there are labels like `"ali.removeMounts": "/etc/entrypoint.sh"`. So we need to add `/`.

Reply:
yes, / is necessary, need to add it.